climbStairs.py

```diff
@@ -0,0 +1,36 @@
+class Solution(object):
+    def climbStairs(self, n):
+        """
+        : type n: int
+        : rtype: int
+        """
+        #second solution
+        # to make solution time complexity O(n)
+        # f(n) = f(n -1) + f(n-2)
+        # use a list to memorize previous solution
+        result = []
+        result.append(1)
+        result.append(1)
+        for i in range(2, n+1):
```

Comment:
which version of python are you using and what's the space complexity of `range` ?